server.js

```diff
@@ -8,7 +8,7 @@ const requestProxy = require('express-request-proxy'); // REVIEW: We've added a
 const PORT = process.env.PORT || 3000;
 const app = express();
 // const conString = 'postgres://USERNAME:PASSWORD@HOST:PORT';
-const conString = ''; // TODO: Don't forget to set your own conString
+const conString = 'postgres://boucher:rowen123@localhost:5432/'; // TODO: Don't forget to set your own conString
```

Comment:
Your articles don't render when clicked through your Heroku link on my machine. I think it has something to do with how you set your constring. The current constring will only work on your local machine. Weren't you guys told in class to set `process.env.DATABASE_URL` as your constring?